Units/parser-javascript.r/js-es6-mixin.d/input.js

```diff
@@ -0,0 +1,28 @@
+
+class Foo {
+  foo() { return "foo"; }
+}
+
+class Bar extends Foo {
+  bar() { return "bar"; }
+}
+
+function Mixin(superclass) {
+  return class extends superclass {
+    hello() {
```

Comment:
Well, the anonymous class here is ignored altogether, because what comes after `return` is skipped.  So… no, ideally it'd be captured I guess, but it's not really a "bug", rather a limitation.  I'll look into it, now we have anonymous classes anyway, it could be mapped to something.